extensions/Worker.Extensions.CosmosDB/src/CosmosDBInputAttribute.cs

```diff
@@ -11,11 +12,34 @@ namespace Microsoft.Azure.Functions.Worker
     [InputConverter(typeof(CosmosDBConverter))]
     public sealed class CosmosDBInputAttribute : InputBindingAttribute
     {
-        /// <summary>
+        /// <Summary>
         /// Constructs a new instance.
+        /// Use this constructor when binding to a CosmosClient.
+        /// </summary>
+        public CosmosDBInputAttribute()
+        {
+            DatabaseName = string.Empty;
```

Comment:
Can an alternative approach be to assign `DefaultValue` attribute as below? https://github.com/Azure/azure-functions-dotnet-worker/blob/9efb07783a1b8769fd87ef4199d356544bf0bf2d/extensions/Worker.Extensions.Storage.Blobs/src/BlobInputAttribute.cs#L32-L33

Reply:
Let me try it out :)

Reply:
Doesn't quiet do the job:

`There is no argument given that corresponds to the required parameter 'databaseName' of 'CosmosDBInputAttribute.CosmosDBInputAttribute(string, string)' [/Users/likasem/source/functions/azure-functions-dotnet-worker/samples/WorkerBindingSamples/WorkerBindingSamples.csproj`